src/components/Footer.astro

```
@@ -36,14 +39,27 @@ const year = new Date().getFullYear();
         <Heading level={3} variant="footer">Kontakt</Heading>
         <ul class="mt-3 space-y-2 text-sm">
           <li class="text-neutral-800">Feldkirchen-Westerham</li>
-          <li><TextLink href="mailto:kontakt@stamm-phoenix.de" variant="footer">kontakt@stamm-phoenix.de</TextLink></li>
+          <li>
+            <TextLink href="mailto:kontakt@stamm-phoenix.de" variant="footer"
+              >kontakt@stamm-phoenix.de</TextLink>
```

ESLint

[error] Insert `⏎············`

(prettier/prettier)